### DISCUSSION TITLE
paid but no code

### DISCUSSION THREAD

**Discussion by huanglei288766** [getlantern/lantern-forum-cn#661] [General]
i paid but no code was sent to my email.

emai: huanglei288766@gmail.com

orderId：2024070122001319941425618328  src_1PXjGYHGmABNLbfYGaFbhNvH

@Mike777Khoo

**Comment by MaxMKMax** [comment]
@huanglei288766 Hi user, in order to protect your privacy information. Can you submit the issues via this link? Provide your email address and payment receipt via the link. Our customer support will help you solve the issues and reply you with the code.

Thank you for supporting Lantern. Appreciate it so much.

https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
